.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on: push
```

Comment:
`name` is the name of the overall workflow. CI might not be a very helpful name here and we could change this as the app gets more developed. We might have a separate workflow for publishing the app for example.

`on` is a way to define what triggers this workflow. `push` will cause this workflow to run for any push to this repo. There's a [long list of events that can trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows) but this should be all we need for a while.